### DISCUSSION TITLE
[Suggestion] Hide unused menu items on Windows

### DISCUSSION THREAD

**Discussion by adevade** [spatie/ray#539] [Ideas]
![image](https://user-images.githubusercontent.com/1066486/128720989-25b9cd0c-e190-4bc3-8c9d-f86fe0bb03fd.png)

I think that `Services`, `Hide Ray`, `Hide Others` and `Show all` are only used in macOS, and could be removed for Windows builds of the client.
I was a bit confused at first, because clicking them doesn't do anything and gives no feedback.

**Comment by freekmurze** [comment]
Makes sense

@AdrianMrn could you take care of this?

**Reply by AdrianMrn** [reply]
This change will be included in the next release!
